.github/workflows/MacOsX.yaml

```diff
@@ -0,0 +1,200 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Continuous integration tests that pull requests are required to pass
+name: MacOsX
```

Comment:
The official name is `macOS` now :) Also remove the `X` from the file name please